src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -0,0 +1,182 @@
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import TableListItem from '@components/SelectionList/TableListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceTaxesPageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.TAXES>;
+
+function WorkspaceTaxesPage({policy, route}: WorkspaceTaxesPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [selectedTaxesIDs, setSelectedTaxesIDs] = useState<string[]>([]);
+    const defaultExternalID = policy?.taxRates?.defaultExternalID;
+    const foreignTaxDefault = policy?.taxRates?.foreignTaxDefault;
+
+    const textForDefault = useCallback(
+        (taxID: string): string => {
+            if (taxID === defaultExternalID && taxID === foreignTaxDefault) {
+                return translate('common.default');
+            }
+            if (taxID === defaultExternalID) {
+                return translate('workspace.taxes.workspaceDefault');
+            }
+            if (taxID === foreignTaxDefault) {
+                return translate('workspace.taxes.foreignDefault');
+            }
+            return '';
+        },
+        [defaultExternalID, foreignTaxDefault, translate],
+    );
+
+    const taxesList = useMemo<ListItem[]>(
+        () =>
+            Object.entries(policy?.taxRates?.taxes ?? {})
+                .map(([key, value]) => ({
+                    text: value.name,
+                    alternateText: textForDefault(key),
+                    keyForList: key,
+                    isSelected: !!selectedTaxesIDs.includes(key),
+                    isSelectable: key !== defaultExternalID,
```

Comment:
Here, we update to `isDisabledCheckbox: key === defaultExternalID,`


One more question, why do you remove `foreignTaxDefault` here? I don't see the condition to disable the checkbox of tax on our document

Reply:
I've added it to the doc and I've removed `foreignTaxDefault`, because tax selected as `foreignTaxDefault` may be removed or disabled in Old Dot, so it can be selected also to bulk mode